CHANGELOG.md

```diff
@@ -11,6 +12,31 @@
 - [v2.15.0-ck8s1](#v2150-ck8s1---2021-05-27)
 <!-- END TOC -->
 
+-------------------------------------------------
+## v2.21.0-ck8s1 - 2023-02-06
+
+### Release notes
+
+- This requires at least terraform 0.14.0
```

Comment:
```suggestion
- This version requires at least terraform 0.14.0 in order to be able to provision infrastructure using the kubespray provided terraform modules.
```

Reply:
I believe this is the wrong place to make changes, I can do it on the QA branch